Amigolden/src/app/pages/events-list/events-list.page.html

```diff
@@ -4,37 +4,51 @@
       <ion-menu-button></ion-menu-button>
     </ion-buttons>
     <ion-buttons slot="end">
-      <button *ngIf="headerOptions.canCreate" (click)="onCreate()" ion-button>
-          <ion-icon name="create"></ion-icon>
-          &nbsp;Create
-      </button>
-    </ion-buttons>    
+      <ion-button
+        size="small"
+        fill="solid"
+        color="primary"
+        *ngIf="headerOptions.canCreate"
+        (click)="onCreate()"
+        >&nbsp;Create<ion-icon name="create"></ion-icon
+      ></ion-button>
+    </ion-buttons>
     <!-- <app-search (searchFilter)="this.config.filter=$event"></app-search> -->
   </ion-toolbar>
 </ion-header>
 <ion-content>
-  <ion-segment #viewController>
-      <ion-segment-button value="map">
-        <ion-label>Map</ion-label>
-      </ion-segment-button>
-      <ion-segment-button checked="true" value="list">
-        <ion-label>List</ion-label>
-      </ion-segment-button>
-    </ion-segment>
-  <app-list-base *ngIf="viewController.value == 'list'" [config]="config" [template]="itemTemplate"></app-list-base>
+  <ion-segment #viewController mode="ios">
+    <ion-segment-button value="map" checked="true">
+      <ion-label>Map</ion-label>
+    </ion-segment-button>
+    <ion-segment-button value="list">
+      <ion-label>List</ion-label>
+    </ion-segment-button>
+  </ion-segment>
+
+  <app-list-base
+    *ngIf="viewController.value == 'list'"
+    [config]="config"
+    [template]="itemTemplate"
+  ></app-list-base>
   <div *ngIf="viewController.value == 'map'">
-      <app-map [options]="mapOptions" [infoWindowTemplate]="mapTemplate" (locationSelected)="navigateToDetail($event.data)" ></app-map>
+    <app-map
+      [options]="mapOptions"
+      [infoWindowTemplate]="mapTemplate"
```

Comment:
the info window template code probably belongs here.  I think the map template is suppose to be injected into the map.component.ts , If its too troublesome to figure out you can leave it and i'll fix it